modules/config.js

```diff
@@ -12,7 +12,10 @@ module.exports = {
     return typeof _matchMediaFunction === 'function';
   },
 
-  matchMedia (query: string) {
+  matchMedia (query: string): {
+    addListener: () => void,
```

Comment:
this should cause flow to fail, but I think it doesn't because flow loses all info on bind. oh well, good enough